npm: No matching version found for @mui/lab@next in CRA styled-components demo

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/mui-org/material-ui/issues) of this repository and believe that this is not a duplicate.

## Current Behavior 😯

npm fails when installing the example for styled components with CreateReactApp (https://github.com/mui-org/material-ui/tree/master/examples/create-react-app-with-styled-components)

`npm ERR! notarget No matching version found for @mui/lab@next.`

## Expected Behavior 🤔

package should be installable/resolve

## Steps to Reproduce 🕹

Follow the instructions in the readme in the link above


## Context 🔦
Demo is not 

## Your Environment 🌎

<!--
  Run `npx @mui/envinfo` and post the results.
  If you encounter issues with TypeScript please include the used tsconfig.
-->

```
  System:
    OS: macOS 11.5.2
  Binaries:
    Node: 14.17.3 - /var/folders/hd/jkczj0nx7zb6xywcm3x9hbn80000gn/T/fnm_multishells/99329_1632422775630/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 7.23.0 - /var/folders/hd/jkczj0nx7zb6xywcm3x9hbn80000gn/T/fnm_multishells/99329_1632422775630/bin/npm
  Browsers:
    Chrome: 94.0.4606.54
    Edge: Not Found
    Firefox: 91.0.2
    Safari: 14.1.2
```

--- a/examples/create-react-app-with-styled-components-typescript/package.json
+++ b/examples/create-react-app-with-styled-components-typescript/package.json
@@ -3,9 +3,9 @@
   "version": "5.0.0",
   "private": true,
   "dependencies": {
-    "@mui/material": "next",
-    "@mui/lab": "next",
-    "@mui/styled-engine-sc": "next",
+    "@mui/material": "latest",
+    "@mui/lab": "latest",
+    "@mui/styled-engine-sc": "latest",
     "@testing-library/jest-dom": "latest",
     "@testing-library/react": "latest",
     "@testing-library/user-event": "latest",

--- a/examples/create-react-app-with-styled-components/package.json
+++ b/examples/create-react-app-with-styled-components/package.json
@@ -3,9 +3,9 @@
   "version": "5.0.0",
   "private": true,
   "dependencies": {
-    "@mui/material": "next",
-    "@mui/lab": "next",
-    "@mui/styled-engine-sc": "next",
+    "@mui/material": "latest",
+    "@mui/lab": "latest",
+    "@mui/styled-engine-sc": "latest",
     "@testing-library/jest-dom": "latest",
     "@testing-library/react": "latest",
     "@testing-library/user-event": "latest",

--- a/examples/create-react-app-with-typescript/package.json
+++ b/examples/create-react-app-with-typescript/package.json
@@ -5,7 +5,7 @@
   "dependencies": {
     "@emotion/react": "latest",
     "@emotion/styled": "latest",
-    "@mui/material": "next",
+    "@mui/material": "latest",
     "@types/react": "latest",
     "@types/react-dom": "latest",
     "react": "latest",

--- a/examples/create-react-app/package.json
+++ b/examples/create-react-app/package.json
@@ -11,7 +11,7 @@
   "dependencies": {
     "@emotion/react": "latest",
     "@emotion/styled": "latest",
-    "@mui/material": "next",
+    "@mui/material": "latest",
     "react": "latest",
     "react-dom": "latest",
     "react-scripts": "latest"

--- a/examples/gatsby/package.json
+++ b/examples/gatsby/package.json
@@ -6,7 +6,7 @@
     "@emotion/react": "latest",
     "@emotion/server": "latest",
     "@emotion/styled": "latest",
-    "@mui/material": "next",
+    "@mui/material": "latest",
     "gatsby": "latest",
     "gatsby-plugin-react-helmet": "latest",
     "react": "latest",

--- a/examples/nextjs-with-styled-components-typescript/package.json
+++ b/examples/nextjs-with-styled-components-typescript/package.json
@@ -9,8 +9,8 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@mui/material": "next",
-    "@mui/styled-engine-sc": "next",
+    "@mui/material": "latest",
+    "@mui/styled-engine-sc": "latest",
     "next": "latest",
     "next-transpile-modules": "latest",
     "react": "latest",

--- a/examples/nextjs-with-typescript/package.json
+++ b/examples/nextjs-with-typescript/package.json
@@ -7,7 +7,7 @@
     "@emotion/react": "latest",
     "@emotion/styled": "latest",
     "@emotion/server": "latest",
-    "@mui/material": "next",
+    "@mui/material": "latest",
     "clsx": "latest",
     "next": "latest",
     "react": "latest",

--- a/examples/nextjs/package.json
+++ b/examples/nextjs/package.json
@@ -7,7 +7,7 @@
     "@emotion/react": "latest",
     "@emotion/styled": "latest",
     "@emotion/server": "latest",
-    "@mui/material": "next",
+    "@mui/material": "latest",
     "clsx": "latest",
     "next": "latest",
     "prop-types": "latest",

--- a/examples/preact/package.json
+++ b/examples/preact/package.json
@@ -6,7 +6,7 @@
     "@babel/helper-builder-react-jsx": "latest",
     "@emotion/react": "latest",
     "@emotion/styled": "latest",
-    "@mui/material": "next",
+    "@mui/material": "latest",
     "@testing-library/jest-dom": "latest",
     "@testing-library/react": "latest",
     "@testing-library/user-event": "latest",

--- a/examples/ssr/package.json
+++ b/examples/ssr/package.json
@@ -12,7 +12,7 @@
     "@emotion/react": "latest",
     "@emotion/styled": "latest",
     "@emotion/server": "latest",
-    "@mui/material": "next",
+    "@mui/material": "latest",
     "babel-loader": "latest",
     "cross-env": "latest",
     "express": "latest",

